.github/pull_request_template.md

```diff
@@ -12,10 +12,10 @@ Please delete options that are not relevant.
 
 # Checklist:
 
-- [ ] My content uses the appropriate readme template
+- [ ] My content uses the appropriate readme [template](https://github.com/Azure/azure-sphere-gallery/tree/main/Templates)
 - [ ] I have performed a self-review of my own contribution
 - [ ] I have provided comments where appropriate
-- [ ] I have made corresponding changes to the README.md and other relevant doc files
-- [ ] I have made the appropriate licensing updates
+- [ ] I have updated the repository's [README.md](https://github.com/Azure/azure-sphere-gallery/blob/main/README.md) to index this project
```

Comment:
Do we want to keep the "and any other relevant documents"? Perhaps as a separate bullet/check to say "No other doc updates needed" or list out other updates?

Reply:
after discussion, not doing this - so that the checklist is specific/actionable